core/class/cmd.class.php

```diff
@@ -1134,10 +1134,12 @@ public function remove() {
 		viewData::removeByTypeLinkId('cmd', $this->getId());
 		dataStore::removeByTypeLinkId('cmd', $this->getId());
 		$eqLogic = $this->getEqLogic();
-		$eqLogic->setStatus(array(
-			'warning' => 0,
-			'danger' => 0,
-		));
+		if (is_object($eqLogic)) {
```

Comment:
&& method_exists($eqLogic, 'setStatus')

Reply:
Bonjour, Désolé mais je ne comprend pas la proposition.
Hello, Sorry but I don't understand the proposal.

Reply:
Tu vérifie que ta variable est un objet. Si c'est le cas tu appels la methode set Status, mais rien n'indique ici que cette méthode existe dans ton objet. Ma proposition permet juste de faire cette vérification.

Reply:
La verification doit suffire dans quasiment tous les cas. Il est peut probable que l’objet n’est pas la méthode en question.